src/Symfony/Component/Console/Tester/CommandTester.php

```diff
@@ -129,4 +137,48 @@ public function getStatusCode()
     {
         return $this->statusCode;
     }
+
+    /**
+     * Sets the user inputs.
+     *
+     * @param array
+     *
+     * @return CommandTester
```

Comment:
Imo this docblock as it is now is quite useless and could be removed as well (input types and the return type can be inferred by IDEs from the method itself) and the description doesn't add any value to the user apart from what the method name already states. Though explaining what `$inputs` exactly is expected to be would be something that justified its presence.